components/homeComponents/HomeSponsors.tsx

```diff
@@ -10,20 +18,28 @@ export default function HomeSponsors(props: { sponsorCard: Sponsor[] }) {
 
   return (
     sponsor.length != 0 && (
-      <section className="md:p-12 p-6">
-        <div className="flex flex-col flex-grow bg-white">
-          <h4
-            // change this after updating tailwind font
-            style={{
-              fontFamily: 'Fredoka',
-              fontWeight: '600',
-            }}
-            className="text-complementary font-bold md:text-4xl text-2xl my-4 text-center"
-          >
-            Our Sponsors
+      <section className="relative pt-[10rem] bg-[#DAC397]">
+        {/* TODO: will update styling better once get more assets and finalized content */}
+        <div>
+          <div className="text-center text-3xl text-white">
+            <h1 className="uppercase">see you there!</h1>
+          </div>
+          <div className="flex flex-wrap w-full items-end justify-center">
```

Comment:
Can we do something like `align-center` to make them align with each other vertically?